webpack.config.js

```diff
@@ -13,7 +15,10 @@ module.exports = {
     "react-dom": "react-dom"
   },
   resolve: {
-    extensions: [".js", ".jsx", ".ts", ".tsx"]
+    extensions: [".js", ".jsx", ".ts", ".tsx"],
+    fallback: {
```

Comment:
why is it different to the one above ? (stream)

Reply:
ok i will make the 2 webpack files consistent on the polyfill